readme.md

```diff
@@ -95,6 +95,14 @@ Type: `array`
 
 Pick which methods in a module **not** to promisify.
 
+##### excludeMain
+
+Type: `array`
+
+*Works for `pify.all()` only.*
+
+Set to `true` if you do **not** want to promisify module's main function.
```

Comment:
Maybe show an example or mention a use-case. I don't think most realize you can have `main` method with additional methods a properties.